cli/src/main/kotlin/edu/cornell/cs/apl/viaduct/cli/Compile.kt

```diff
@@ -155,7 +166,20 @@ class Compile : CliktCommand(help = "Compile ideal protocol to secure distribute
         )
         val postprocessedProgram = postprocessor.postprocess(annotatedProgram)
 
-        output.println(postprocessedProgram)
+        if (compileKotlin) {
+
+            // TODO - figure out best way to let code generators know which protocols it is responsible for
```

Comment:
The `ProtocolInterpreter` interface has a member `availableProtocols` that define the protocols it is supposed to interpret; you can probably do the same thing here